exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/trace_exporter/__init__.py

```diff
@@ -40,6 +40,7 @@
 from opentelemetry.proto.trace.v1.trace_pb2 import Span as CollectorSpan
 from opentelemetry.proto.trace.v1.trace_pb2 import Status
 from opentelemetry.sdk.environment_variables import (
+    OTEL_EXPORTER_OTLP_INSECURE,
```

Comment:
This should be `OTEL_EXPORTER_OTLP_TRACES_INSECURE`. Right now the env is `*_SPAN_*` but there is a spec PR to update this https://github.com/open-telemetry/opentelemetry-specification/pull/2240

Reply:
Added. I have added a `if not credentials:` check in the base exporter to not overwrite credentials if some are passed in. This will give `OTEL_EXPORTER_OTLP_TRACES_CERTIFICATE` precedence.